### PR TITLE
Added instagram integration, on instagram links shows title and image

### PR DESCRIPTION
**Pull request by Stanko**
Instagram integration, I reused most of the logic for the tweets.
Tweaked CSS for tweets (and istangram) a bit.

instagram

### CODE REVIEW

File: src/ui/css/yakyak/colors.less [hunk 1]
```diff
@@ -38,6 +40,7 @@ html{
 	    --darkred: #804cad;
 	    --link: #F9C15C;
 	    --tweetbg: #644104;
+      --instagrambg: #333940;
```
Comment (davibe) [inline src/ui/css/yakyak/colors.less]: different indentation ?

File: src/ui/css/yakyak/messages.less [hunk 2]
```diff
@@ -56,8 +56,9 @@
                     padding-left: 5px;
                     padding-right: 30px;
                     background: var(--altgrey);
-                    .message .tweet{
-                        margin: 10px -34px -5px -10px;
+                    .message .tweet,
+                    .message .instagram{
```
Comment (davibe) [inline src/ui/css/yakyak/messages.less]: add space before `{`

File: src/ui/css/yakyak/colors.less [hunk 3]
```diff
@@ -63,5 +66,6 @@ html{
...
```
Comment (davibe) [inline src/ui/css/yakyak/colors.less]: as well

**Comment by Stanko** [comment]
@davibe thanks, I'll clean those up :)

Btw some of the less files have mixed spaces and tabs, I didn't want to reformat whole files.

**Comment by Stanko** [comment]
Fixed and squashed.

I reformatted `colors.less` as it was mixed tabs and spaces.

**Comment by arnriu** [comment]
try this:

colors.less:
```
html{
--instaBgYellow: #fed373;
--instaBgRed: #f15245;
--instaBgPink: #d92e7f;
--instaBgPurple: #9b36b7;
--instaBgBlue: #515ecf;
}
```

messages.less
```
.instagram p{
    background: radial-gradient(circle at 0 110%, var(--instaBgYellow) 4%, var(--instaBgRed) 30%, var(--instaBgPink) 60%, var(--instaBgPurple) 85%, var(--instaBgBlue));
}
```

Also, can you add this fix for both tweets imgs and insta imgs:

```
.tweet img{
  width: 100%
}
```

(and also, in messages.less on .tweet, there is a useless background-color just before background).

**Comment by Stanko** [comment]
I've added image width fix, but personally I think that that background is a way too much. It is hard to read, and imagine 4-5 instagram posts one after another.

**Comment by davibe** [comment]
the github thing tells me there are conflicts

**Comment by Stanko** [comment]
@davide rebased, and solved conflicts.

**Comment by arnriu** [comment]
@Stanko Well, you're right, yellow is too bright on the bottom left corner. I just used the new instagram colors, but we could try something less yellowish.
I still think instagram blocks need their own identity different than twitter. What do you think of the header's gradient from https://www.instagram-brand.com ?

**Comment by Stanko** [comment]
Personally, I am against gradients beneath the text, so I would always leave an option to keep it solid colored 😃

**Pull request merged**